### PR TITLE
PLANET-6676 Update proper textdomain name in JS files

### PR DESCRIPTION
**Pull request by sagarsdeshmukh**
Ref. https://jira.greenpeace.org/browse/PLANET-6676

- Remove translation of block name, block style name etc.
- Update text-domain name where-ever needed.

### CODE REVIEW

File: assets/src/blocks/Accordion/AccordionEditorScript.js [hunk 1]
```diff
@@ -37,9 +37,9 @@ registerBlockType(BLOCK_NAME, {
   icon: 'menu',
   category: 'planet4-blocks',
   keywords: [
-    __('accordion'),
-    __('faq'),
-    __('collapsible')
+    'accordion',
```
Comment (mleray) [inline assets/src/blocks/Accordion/AccordionEditorScript.js:40]: These don't need translations? 🤔

Comment (sagarsdeshmukh) [reply assets/src/blocks/Accordion/AccordionEditorScript.js:40]: As discussed in sprint meeting, we are excluding the block keywords from translation.

**Review by mleray — APPROVED** [review]
Looks good to me 👍

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [iocaste](https://www-dev.greenpeace.org/test-iocaste) | [admin](https://www-dev.greenpeace.org/test-iocaste/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-iocaste/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-iocaste) | [composer-local.json](https://github.com/greenpeace/planet4-test-iocaste/blob/main/composer-local.json)

:watch: 2022.08.19 14:13:27

**Comment by sagarsdeshmukh** [comment]
Hi @mleray - I updated the PR after last review, could you please have a look on new changes?

File: assets/src/blocks/Cookies/CookiesFrontend.js [hunk 2]
```diff
@@ -155,7 +155,7 @@ export const CookiesFrontend = props => {
                 editable={isEditing}
                 allowedFormats={[]}
               />
-              <span className="always-enabled">{__('Always enabled', 'planet4-master-theme')}</span>
+              <span className="always-enabled">{__('Always enabled', 'planet4-blocks-backend')}</span>
```
Comment (mleray) [inline assets/src/blocks/Cookies/CookiesFrontend.js:158]: I think this one should not use the backend domain 🤔

Comment (sagarsdeshmukh) [reply assets/src/blocks/Cookies/CookiesFrontend.js]: Yes, you are right 👍

**Review by mleray — APPROVED** [review]
Looks good to me 👍

**Pull request merged**